js/apps/events-page/controllers/eventsPageCtrl.js

```diff
@@ -0,0 +1,39 @@
+((app) => {
+	'use strict';
+
+	const EventsPageCtrl = function EventsPageCtrl($scope, $timeout, CONSTANTS, eventsService, dateUtility) {
```

Comment:
Do we care that this isn't es6-ish?

Reply:
You can't use arrow syntax here, since I need access to "this", which is undefined in arrow functions.